src/control/cmd/dmg/system.go

```diff
@@ -82,3 +86,47 @@ func (cmd *systemMemberQueryCmd) Execute(args []string) error {
 
 	return nil
 }
+
+// systemListPoolsCmd represents the command to fetch a list of all DAOS pools in the system.
+type systemListPoolsCmd struct {
+	logCmd
+	connectedCmd
+	cfgCmd
+}
+
+// Execute is run when systemListPoolsCmd activates
+func (cmd *systemListPoolsCmd) Execute(args []string) error {
+	if cmd.config == nil {
+		return errors.New("No configuration loaded")
+	}
+	req := client.ListPoolsReq{SysName: cmd.config.SystemName}
+	resp, err := cmd.conns.ListPools(req)
+	if err != nil {
+		cmd.log.Infof("List-Pools command failed: %s\n", err.Error())
+		return err
+	}
+
+	cmd.log.Info("List-Pools command succeeded\n")
+	if len(resp.Pools) == 0 {
+		cmd.log.Info("No pools in system\n")
+		return nil
+	}
+
+	for _, pool := range resp.Pools {
+		var b strings.Builder
+
+		b.WriteString(pool.UUID)
+		if len(pool.SvcReplicas) > 0 {
+			b.WriteString(" ")
+			for i, rep := range pool.SvcReplicas {
+				if i != 0 {
+					b.WriteString(":")
+				}
+				fmt.Fprintf(&b, "%d", rep)
+			}
+		}
+		b.WriteString("\n")
```

Comment:
This output is a little hard to parse/understand. Maybe incorporate tabwriter as @tanabarr is doing in his PR to improve storage scan output? That will allow you to present a nice table-based display with column names. e.g. https://github.com/daos-stack/daos/pull/1495/files#diff-bd69096e8b80741d25d70e29f5d87a6eR124

Reply:
I was going to ask about this actually - do we want some standardized outputter in DMG that accepts, say, a list of dictionaries with the column names as keys, and displays them in properly tabbed columns? If we find ourselves repeating the same display code in multiple places.

Reply:
That's a great idea... Does it make sense to do that as part of this PR? I'd support that because sometimes it's hard to develop that sort of thing unless you have a specific use case in mind, but I wouldn't insist on it for landing this one.

Reply:
I'll look into it and see what it takes. It's the kind of thing I'd normally break into another PR, but if it's not big, it may be worth it to add it now.

Reply:
I went ahead and added a formatter. Based it on the logic in Tom's patch.